particles/Tutorial/Javascript/README.md

```diff
@@ -0,0 +1,52 @@
+Welcome to developing using Arcs! By using Arcs as the platform for your system, you are choosing to value user sovereignty and leverage AI.  To get started, please obtain the project through [Github](https://github.com/PolymerLabs/arcs) using the instructions available there. For debugging purposes, we highly recommend having the Chrome Dev Tools also installed.
```

Comment:
Chrome Dev Tools is devtools of chrome - you get it installed when you install Chrome.

I assume you meant our chrome extension to chrome devtools.

maybe:

For debugging purposes, we highly recommend having the [Arcs Explorer](https://github.com/PolymerLabs/arcs/tree/master/devtools) Chrome Extension also installed.